src/components/Nav.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+	 <b-nav
+	 	class="px-2"
+		:style="styleVariables"
+	 >
+		<b-nav-item
+			v-for="(item, index) in items"
+			:id="getElementKey(item, index)"
+			:key="getElementKey(item, index)"
+			:active="isActive(item)"
+			class="mr-4 mt-2"
+			@click="handleClick(item)"
+		>
+			{{ item.label }}
+		</b-nav-item>
+	</b-nav>
+</template>
+
+<script>
+export default {
+	props: {
+		items: {
+			type: Array,
+			default: () => [],
+			description: 'Uma lista de items que serão exibidos',
+			required: true,
+		},
+		oldSchool: {
+			type: Boolean,
+			default: false,
+			description: 'Define o modo de exibição/sistema de cores do menu',
+		},
+	},
+
+	data() {
+		return {
+			activeItem: this.items[0],
+		};
+	},
+
+	computed: {
+		styleVariables() {
+			if(this.oldSchool) {
+				return {
+					'--active-color': 'rgb(58, 223, 124)',
```

Comment:
It is a bad practice to insert constants inside of functions, this styles should be props. For instance, Bios Project's main color is purple, this constants would make this component incompatible with Bios Project style guides.

Reply:
It was implemented that way because of the bind between the "knobs" addon, that allows us to test the component visualy, and the props that manage color inside the component and that passes the selected color as a variable to the css. This pattern is already been used in our components in the component library.

Reply:
What it I want to use this component with another colors?

Reply:
One of the purposes of the Styleguide we are building is to minimize the inconsistencies in the project and to create specific patterns for the projects. Thus these are the only options we are providing for customization, and that can eventually become only one.

Reply:
So, in the near future this project will not be anymore a Component Library and will become CS Styleguide?

Reply:
The ideia is to keep using the Storybook as a component library. The Style guide will be created in Figma and will determine the patterns followed by the components here. As we are finishing the Bios project, eventually, almost all components will be from the "Cidade Saudável"